apiserver/gui.go

```diff
@@ -140,9 +136,9 @@ func (gr *guiRouter) ensureFiles(req *http.Request) (rootDir string, hash string
 	if err != nil {
 		return "", "", errors.Trace(err)
 	}
-	logger.Debugf("serving Juju GUI version %s", vers)
+	logger.Tracef("serving Juju Dashboard version %s", vers)
```

Comment:
Changed the Debug to Trace as we don't want to see this all the time in the logs. Also renamed GUI to dashboard in many places.